extensions/wikia/Helios/User.class.php

```diff
@@ -252,6 +252,11 @@ public static function onUserLogout() {
 		return true; // So that wfRunHooks evaluates to true.
 	}
 
+	public static function onUserInvalidateToken() {
+        self::invalidateAccessTokenInHelios();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)